lib/new_relixir/statman.ex

```diff
@@ -0,0 +1,186 @@
+defmodule NewRelixir.Statman do
```

Comment:
We don't need to call it `Statman` anymore.
I think we can call it `Stats`, or `StatsAggregator` or another better name.

Reply:
Done in 03942240549525629e2a5862222f61e82908ea27